test/c_glib/src/test_client.c

```diff
@@ -181,11 +194,22 @@ main (int argc, char **argv)
   sigpipe_action.sa_flags = SA_RESETHAND;
   sigaction (SIGPIPE, &sigpipe_action, NULL);
 
+  if (ssl) {
+    thrift_ssl_socket_initialize_openssl();
+  }
+
   /* Establish all our connection objects */
-  socket = g_object_new (THRIFT_TYPE_SOCKET,
+  socket = g_object_new (socket_type,
                          "hostname", host,
                          "port",     port,
+			 "ssl_accept_selfsigned", TRUE,
                          NULL);
+  if (ssl && !thrift_ssl_load_cert_from_file(THRIFT_SSL_SOCKET(socket), "../keys/client.crt")) {
```

Comment:
My mistake - I added that line of code in my first pull request and it caused the next issue to occur.